### DISCUSSION TITLE
Cannot find module '@llamaindex/env/type'

### DISCUSSION THREAD

**Discussion by GeorgeOrzanescu** [run-llama/LlamaIndexTS#577] [Q&A] [answered]
Working on a typescript project and encountered this error when trying to use the example from docs:

`import { HuggingFaceEmbedding, serviceContextFromDefaults } from "llamaindex";

const huggingFaceEmbeds = new HuggingFaceEmbedding();

const serviceContext = serviceContextFromDefaults({ embedModel: openaiEmbeds });

const document = new Document({ text: essay, id_: "essay" });

const index = await VectorStoreIndex.fromDocuments([document], {
  serviceContext,
});

const queryEngine = index.asQueryEngine();

const query = "What is the meaning of life?";

const results = await queryEngine.query({
  query,
});`

**_error TS2307: Cannot find module '@llamaindex/env/type' or its corresponding type declarations.

1 import type { GenericFileSystem } from "@llamaindex/env/type";_**

**Comment by himself65** [comment]
What's your ts-config settings?

**Comment by himself65** [comment]
I would suggest you to change moduleResolution to "bundler"

**Reply by himself65** [reply, marked as answer]
This could help https://github.com/run-llama/LlamaIndexTS/issues/578
